go.mod

```diff
@@ -1,9 +1,9 @@
 module istio.io/client-go
 
-go 1.12
+go 1.16
```

Comment:
This is only a recommendation. Users aren't broken if they don't use the same version of go. In the istio/istio repo developers are permitted to use go 1.16 features. This should not break anyone.